swift/example_code/s3/ListBuckets-Simple/Package.swift

```diff
@@ -0,0 +1,33 @@
+// swift-tools-version: 5.5
+// The swift-tools-version declares the minimum version of Swift required to
+// build this package.
+//
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
(nit) AIUI, the copyright header must be line 1

Reply:
Can't be line 1 in this particular file because the // swift-tools-version line must be first for the package file to work. But I'll move the copyright up to be right under that, and will update the position in the other file in this sample.